public/js/main.js

```diff
@@ -0,0 +1,96 @@
+/*page preloader*/
+function PreLoader(){
+    setTimeout(()=>{
+        $('.page-loader').addClass('d-none')
+    },1000)
+}
+
+
+
+/* banner sliders*/
+$(document).ready(function () {
+    PreLoader()
+    $(".owl-carousel").owlCarousel();
```

Comment:
Move this section to **homepage.js** and change the identified class name to a specific one like **home_banner**
becuase owl-carousel is a global name and it will enable slide for all other owl-carousel of that page also.

Reply:
Fixed